PowerShell/JumpCloud Module/Docs/Set-JCSettingsFile.md

```diff
@@ -14,7 +14,7 @@ Updates the JumpCloud Module Settings File
 ## SYNTAX
 
 ```
-Set-JCSettingsFile [-parallelOverride <PSObject>] [-moduleBannerMessageCount <PSObject>] [<CommonParameters>]
+Set-JCSettingsFile [-moduleBannerMessageCount <PSObject>] [-parallelOverride <PSObject>] [<CommonParameters>]

```

Comment:
I think if we start to add positions to our parameters, it'll prevent these help files from being changed. I can get a card for that.